lib/tasks/setup_calls.rake

```diff
@@ -0,0 +1,6 @@
+namespace :phone_calls do
+  desc 'Adds the phone calls to activejob'
+  task setup: :environment do
+    CallSetupService.new.all
```

Comment:
DYM `.call`?

Reply:
damn, I probably removed the `c` before the commit. I'll correct it